week1_HW/todo-app/src/components/task-list.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import Tasks from "./task"
+
+export default class TodoList extends Component{
+    render() {
+        return (
+            <ul>
+                <Tasks todo="Attend HYF less" taskTime="2018-03-04" />
```

Comment:
You have your `todo.json` file, but you aren't using it yet. Instead you're rendering components out statically here. That's fine for week 1, but look into rendering components in a loop for week 2.